tests/testthat/_snaps/bind.md

```diff
@@ -181,7 +203,7 @@
       # Integers as rows
       suppressMessages(with_memory_prof(vec_rbind_list(ints)))
     Output
-      [1] 2.53KB
+      [1] 22.1KB
```

Comment:
This was a little surprising. This is generating 100 name repair messages and shouldn't be a very common case (it generally indicates the user has messed something up), but the code is a lot slower because of this too.